packages/studio-ui/src/components/CollapsibleSidebar.tsx

```diff
@@ -11,33 +11,43 @@ export default function CollapsibleSidebar({
 }: PropsWithChildren<CollapsibleSidebarProps>): JSX.Element {
   const [isOpen, setIsOpen] = useState(true);
   const toggleIsOpen = useCallback(() => setIsOpen((isOpen) => !isOpen), []);
-
-  const containerStyle = classNames("flex flex-col", {
+  const arrowRight =
+    (side === "right" && isOpen) || (side === "left" && !isOpen);
+  const containerStyle = classNames("flex flex-row", {
```

Comment:
nit: I don't think we should need `flex-row` since that's the default for `flex`